src/molecules/calendar/calendar.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import { Meta, StoryObj } from '@storybook/react';
+import Calendar from '.';
+
+export default {
+  title: 'molecules/calendar',
+  component: Calendar,
+  parameters: {
+    layout: 'centered',
+  },
+  argTypes: {
+    presets: {
+      control: {
+        type: 'boolean',
+      },
+    },
+    fromYear: {
+      control: { type: 'number', min: 1900, max: 2050 },
+    },
+    toYear: {
+      control: { type: 'number', min: 1900, max: 2050 },
+    },
+    type: {
+      control: 'inline-radio',
+      options: ['buttons', 'dropdown', 'dropdown-buttons'],
+    },
+  },
+  args: {
+    presets: false,
+    range: false,
+    view: 'single',
+    className: 'w-full',
+    fromYear: new Date().getFullYear() - 5,
+    toYear: new Date().getFullYear(),
+    type: 'buttons',
+  },
+} as Meta<typeof Calendar>;
+
+export const Template: StoryObj<typeof Calendar> = {
+  args: {
+    presets: true,
+    range: true,
+    view: 'multiple',
+  },
+  // const [filteredValue, setFilteredValue] = useState<string>('');
+  render: (args) => (
+    <Calendar
+      {...args}
+      initialFocus
+      // onSelect={(value) => {
+      //   if ('toISOString' in value) {
+      //     setFilteredValue(value?.toISOString() || '');
+      //   }
+      // }}
+      // selected={filteredValue ? new Date(filteredValue) : undefined}
+    />
```

Comment:
We had the onSelect previously to get the state of the calendar externally, with the new implementation how can we achieve that? 

we have used it as well in the table/details-filters example, to understand how we expect it to work, the idea here is to be able to use this component with other components and get the value of it externally.